.travis.yml

```diff
@@ -10,21 +11,12 @@ cache:
   - yarn: true
   - directories:
     - node_modules
-env:
-  # Required to compile native modules for Node.js v4+
-  # See https://docs.travis-ci.com/user/languages/javascript-with-nodejs#Node.js-v4-(or-io.js-v3)-compiler-requirements
-  - CXX=g++-4.8
```

Comment:
Ubuntu Trusty is now the default on Travis CI so this is no longer required as the gcc version is 4.8.4

Reply:
Thanks!